README.md

```diff
@@ -27,8 +27,7 @@ For the TEI Header, the conversion is roughly based on the [*DTA base format*](h
 
 ## Installation
 
-`mets-mods2tei` is implemented in Python 3. In the following, we assume a working Python 3
-(tested versions 3.5, 3.6 and 3.7) installation.
+`mets-mods2tei` is implemented in Python 3. In the following, we assume a working Python 3 (tested versions 3.9, 3.10 and 3.11) installation.
```

Comment:
no need to statically mention the versions here – should be enough to have it in the dynamic badge above

(speaking of which, we would then need a new badge for GH Actions)

```suggestion
`mets-mods2tei` is implemented in Python 3. In the following, we assume a working Python 3 installation.
```